App/Course App/Resources/GlobalConstants.swift

```diff
@@ -7,15 +7,21 @@
 
 import Foundation
 
-enum Constants {
+enum GlobalConstants {
```

Comment:
I would suggest to avoid global constants. It can be valid in some cases but better to have constants at places in which they relate to. Most of the constants are UI so they can be part of design system. Another can be just for delays in some extension over dispatchqueue etc.